src/machine/board_circuitplay_express.go

```diff
@@ -0,0 +1,70 @@
+// +build sam,atsamd21,circuitplay_express
+
+package machine
+
+// GPIO Pins
+const (
+	D0  = PB09
+	D1  = PB08
+	D2  = PB02
+	D3  = PB03
+	D4  = PA28
+	D5  = PA14
+	D6  = PA05
+	D7  = PA15
+	D8  = PB23
+	D9  = PA06
+	D10 = PA07
+	D11 = 0xff // does not seem to exist
```

Comment:
Wouldn't it be better to leave this one out entirely? It seems to be here only to not break existing Arduino code entirely. We don't need to be that strictly compatible with Arduino code.

Reply:
Seems that identifier is already in the machine file. This is there to avoid breaking the compile. Probably in a future PR that can be changed in both places.

Reply:
Weird. Well let's keep it there, then. Perhaps it does have a pin assigned on a different board?